OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -55,7 +55,7 @@ public class UpdatePath
 
 			new UpdatePath("release-20200503", new UpdateRule[]
```

Comment:
I believe this should be `new UpdatePath("release-20200503", "playtest-20201213", new UpdateRule[]` and then `new UpdatePath("playtest-20201213", new UpdateRule[]` below?

Reference from the last prep branch:

https://github.com/OpenRA/OpenRA/blob/1df3e282533093e407e73df9ade96c0495545258/OpenRA.Mods.Common/UpdateRules/UpdatePath.cs#L64-L87

Reply:
Otherwise LGTM, so if you agree with these changes feel free to immediately merge after updating.

Reply:
Fixed, I don't actually know why I used the branch name here.